lib/OpenQA/CacheService/Model/Cache.pm

```diff
@@ -70,8 +70,8 @@ sub init ($self) {
     unless (-e $db_file) {
         $log->info(qq{Creating cache directory tree for "$location"});
         $location->remove_tree({keep_root => 1});
-        $location->child('tmp')->make_path;
     }
+	$location->child('tmp')->make_path;
```

Comment:
Maybe 
```suggestion
	my $tmp = $location->child('tmp'); 
	$tmp->make_path unless $tmp->exists;
```
Why to call `make_path` if it does exist

Reply:
> Why to call make_path if it does exist

Because it will check if the dir is there and create it if needed. No need to explicitly code that.


> you might also need to check whether is dir or not.

If someone actually creates a file there called `tmp`, it will raise an error which is what we want in this case.

Reply:
you guys are right. I just check that to make sure.